quickwit-cli/src/cli.yaml

```diff
@@ -92,6 +92,11 @@ subcommands:
                 long: index-uri
                 value_name: INDEX URI
                 required: true
+            - grace-period:
+                help: Threshold period after which intermediate files can be garbage collected.
```

Comment:
i did not know the term. I suggest not making that configurable though. 
1h is a good one size fits all parameter.